mmv1/products/gemini/DataSharingWithGoogleSetting.yaml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: DataSharingWithGoogleSetting
+description: The resource for managing DataSharingWithGoogle settings for Admin Control.
+min_version: 'beta'
+base_url: projects/{{project}}/locations/{{location}}/dataSharingWithGoogleSettings
+update_mask: true
+self_link: projects/{{project}}/locations/{{location}}/dataSharingWithGoogleSettings/{{data_sharing_with_google_setting_id}}
+create_url: projects/{{project}}/locations/{{location}}/dataSharingWithGoogleSettings?dataSharingWithGoogleSettingId={{data_sharing_with_google_setting_id}}
+update_verb: PATCH
+id_format: projects/{{project}}/locations/{{location}}/dataSharingWithGoogleSettings/{{data_sharing_with_google_setting_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/dataSharingWithGoogleSettings/{{data_sharing_with_google_setting_id}}
+mutex: projects/{{project}}/locations/{{location}}/dataSharingWithGoogleSettings/{{data_sharing_with_google_setting_id}}
+examples:
+  - name: gemini_data_sharing_with_google_setting_basic
+    min_version: 'beta'
+    primary_resource_id: example
```

Comment:
You can add `exclude_test: true` here too - in this case the test is passing but since it's the same as `testAccGeminiDataSharingWithGoogleSetting_geminiDataSharingWithGoogleSettingBasicExample_basic` it's not testing anything new.